storm-webapp/src/main/java/org/apache/storm/daemon/logviewer/handler/LogviewerLogPageHandler.java

```diff
@@ -61,18 +63,21 @@
 import org.apache.commons.lang.StringUtils;
 import org.apache.storm.daemon.logviewer.LogviewerConstant;
 import org.apache.storm.daemon.logviewer.utils.DirectoryCleaner;
+import org.apache.storm.daemon.logviewer.utils.ExceptionMeters;
 import org.apache.storm.daemon.logviewer.utils.LogviewerResponseBuilder;
 import org.apache.storm.daemon.logviewer.utils.ResourceAuthorizer;
 import org.apache.storm.daemon.logviewer.utils.WorkerLogs;
 import org.apache.storm.daemon.ui.InvalidRequestException;
 import org.apache.storm.daemon.ui.UIHelpers;
 import org.apache.storm.daemon.utils.StreamUtil;
 import org.apache.storm.daemon.utils.UrlBuilder;
+import org.apache.storm.metric.StormMetricsRegistry;
 import org.apache.storm.utils.ConfigUtils;
 import org.apache.storm.utils.ServerUtils;
 import org.jooq.lambda.Unchecked;
 
 public class LogviewerLogPageHandler {
+    private static final Meter numPageRead = StormMetricsRegistry.registerMeter("logviewer:num-page-read");
```

Comment:
What is this metric useful for?

Reply:
I mean why does someone want to know this :P. Nevermind, I'm guessing this was also requested by @kishorvpatil. I just wanted to know whether someone asked for/had a use case for these metrics.

Reply:
It's as part of IO performance metrics. He seems to be concerning about the IO performance of logviewer, which makes sense to me because majority of our API calls are blocking call.